src/test/java/nl/tudelft/ewi/devhub/modules/MockedGitoliteGitServerModule.java

```diff
@@ -126,7 +126,6 @@ protected void createMockedMirrorsFolder() {
 	protected void createMockedRepositoriesFolder() {
 		String repositoriesPath = repositoriesFolder.toPath().toString() + "/";
 		when(configuration.getGitoliteBaseUrl()).thenReturn(repositoriesPath);
-		when(configuration.getRepositoriesDirectory()).thenReturn(repositoriesFolder);
```

Comment:
Leaving it in now will break the tests for strict stubbing. We can use the silent version if you want to? Still, I think it can be removed for now?